lstchain/irf/hdu_table.py

```diff
@@ -0,0 +1,424 @@
+import numpy as np
+import logging
+
+import astropy.units as u
+from astropy.table import Table, QTable
+from astropy.io import fits
+from astropy.coordinates import SkyCoord, EarthLocation, AltAz
+from astropy.time import Time
+
+from lstchain.__init__ import __version__
+
+__all__ = ["create_obs_index_hdu", "create_hdu_index_hdu", "create_event_list"]
+
+log = logging.getLogger(__name__)
+
+DEFAULT_HEADER = fits.Header()
+DEFAULT_HEADER["CREATOR"] = f"lstchain v{__version__}"
+DEFAULT_HEADER["HDUDOC"] = "https://github.com/open-gamma-ray-astro/gamma-astro-data-formats"
+DEFAULT_HEADER["HDUVERS"] = "0.2"
+DEFAULT_HEADER["HDUCLASS"] = "GADF"
+DEFAULT_HEADER["ORIGIN"] = "CTA"
+DEFAULT_HEADER["TELESCOP"] = "CTA-N"
+DEFAULT_HEADER["CREATED"] = Time.now().utc.iso
+
+location = EarthLocation.from_geodetic(-17.89139 * u.deg, 28.76139 * u.deg, 2184 * u.m)
```

Comment:
Isn't this already defined somewhere in lstchain?

Reply:
Yes. I import it now from `lstchain.reco.utils`